lib/alephant/logger/json.rb

```diff
@@ -9,24 +9,20 @@ def initialize(log_path, options = {})
         @nesting = options.fetch(:nesting, false)
       end
 
-      private
-
-      def flatten_values_to_s(hash)
-        Hash[hash.map { |k, v| [k, v.to_s] }]
-      end
-
-      public
-
       [:debug, :info, :warn, :error].each do |level|
         define_method(level) do |hash|
+          return unless hash.is_a? Hash
```

Comment:
@revett Much better than catching all `IndexError`s

If we're specifically trying to counter `String`s, I'd check `if hash.is_a? String` - reason being: we wouldn't want to forbid users from passing subclasses of `Hash` or common alternatives like `Hashie::Mash`

Just for sake of posterity: _do_ believe we should add a `String` message into a `Hash` and log it in order to meet the clear intention of the client, if we're not going to otherwise alert/error on this that is.  Don't want to bikeshed on that issue though :+1: